demo.py

```diff
@@ -34,4 +40,13 @@
         parser.parse("abcdefg", document)
     except ApplicationError:
         pass
+
+    bob = person.Person("Bob", 42)
+    g1 = bob.introduce()
+    
+    bob.age = 43
```

Comment:
If the class has not defined its own `__setattr__`, the behievor would be `self.__dict__[attr] = value`, where the `self.__dict__` is immutable for us to instrument